torch/csrc/distributed/rpc/tensorpipe_utils.cpp

```diff
@@ -326,6 +327,11 @@ c10::intrusive_ptr<Message> tensorpipeDeserialize(
           ", but got it on ",
           tensors[i].device());
     }
+    if (tensors[i].is_sparse()) {
+      tpDescriptorPos += 2;
+    } else {
+      tpDescriptorPos += 1;
+    }
```

Comment:
will it be safer if we test both indices and values tensor descriptors, instead of just testing values?

Reply:
Testing both is unnecessary because if the first fails, then the second falls. The additional tensor descriptor for sparse tensors does not change the order that the descriptors are sent.

Tensors -> (dense1,dense2,sparse3)
Descriptors -> (dense1,dense2,sparse3,sparse3)